etc/shadow-maint/userdel-pre.d/01-kill_user_procs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Check user exists, and if so, send sigkill to processes that the user owns
+
+RUNNING=`/bin/ps -u "${SUBJECT}" 2>/dev/null | /usr/bin/wc -l`
```

Comment:
I'm worried about the availability of 'ps -u'.  For instance, 'busybox ps' doesn't seem to support it.